### DISCUSSION TITLE
resend cors error

### DISCUSSION THREAD

**Discussion by ican0220** [community/community#141824] [Programming Help]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Hello, super coders,
I am vlady. I am facing to a cors problem. My next.js project is using resend module to send email. But when I use resend api and run project, cors error occurs. Please teach me the reason kindly.

**Comment by ican0220** [comment]
Good answer. Email is sent successfully after I followed you.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @ican0220 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
